### ISSUE TITLE
Implement a way to send commands (wine, 

### ISSUE THREAD

**Issue by qparis** [PhoenicisOrg/phoenicis#1601]
### Expected behavior
* In POLv4, when you open a terminal inside a prefix, you can use the POL 

### System information
* software version (git revision) of phoenicis and scripts repositories
* used operating system
* used Java version

**Comment by madoar** [comment]
Can you give us a list of things that should be doable from inside a POL command line/terminal?

**Comment by plata** [comment]
This is a whole bunch of features. Again something which would be easier to track with Zen Hub epics (in my opinion). What do you think?

**Comment by plata** [comment]
No, it's inside issues: https://www.zenhub.com/blog/working-with-epics-in-github/

**Comment by qparis** [comment]
Ok great!

**Comment by plata** [comment]
I'll activate it. If it doesn't fit our needs, we can get rid of it again.

**Comment by plata** [comment]
Mh. You need the browser extension such that it works properly. @qparis @madoar please give it a try to see if this really makes sense for us (I thought installing the GitHub app would be enough).
